packages/@aws-cdk/aws-stepfunctions-tasks/lib/athena/start-query-execution.ts

```diff
@@ -167,25 +174,46 @@ export class AthenaStartQueryExecution extends sfn.TaskStateBase {
    * @internal
    */
   protected _renderTask(): any {
-    return {
-      Resource: integrationResourceArn('athena', 'startQueryExecution', this.integrationPattern),
-      Parameters: sfn.FieldUtils.renderObject({
-        QueryString: this.props.queryString,
-        ClientRequestToken: this.props.clientRequestToken,
-        QueryExecutionContext: {
-          Catalog: this.props.queryExecutionContext?.catalogName,
-          Database: this.props.queryExecutionContext?.databaseName,
-        },
-        ResultConfiguration: {
-          EncryptionConfiguration: {
-            EncryptionOption: this.props.resultConfiguration?.encryptionConfiguration?.encryptionOption,
-            KmsKey: this.props.resultConfiguration?.encryptionConfiguration?.encryptionKey,
+    if (this.props.resultConfiguration?.outputLocation) {
+      return {
+        Resource: integrationResourceArn('athena', 'startQueryExecution', this.integrationPattern),
+        Parameters: sfn.FieldUtils.renderObject({
+          QueryString: this.props.queryString,
+          ClientRequestToken: this.props.clientRequestToken,
+          QueryExecutionContext: {
+            Catalog: this.props.queryExecutionContext?.catalogName,
+            Database: this.props.queryExecutionContext?.databaseName,
           },
-          OutputLocation: this.props.resultConfiguration?.outputLocation,
-        },
-        WorkGroup: this.props.workGroup,
-      }),
-    };
+          ResultConfiguration: {
+            EncryptionConfiguration: {
+              EncryptionOption: this.props.resultConfiguration?.encryptionConfiguration?.encryptionOption,
+              KmsKey: this.props.resultConfiguration?.encryptionConfiguration?.encryptionKey,
+            },
+            OutputLocation: 's3://' + this.props.resultConfiguration?.outputLocation?.bucketName + '/' + this.props.resultConfiguration?.outputLocation?.objectKey + '/',
```

Comment:
prefer string interpolation to concatenation
```suggestion
            OutputLocation: `s3://${this.props.resultConfiguration?.outputLocation?.bucketName}/${this.props.resultConfiguration?.outputLocation?.objectKey}/`,
```